tests/functional/aws-node-sdk/test/bucket/deleteWebsite.js

```diff
@@ -0,0 +1,45 @@
+import assert from 'assert';
+
+import withV4 from '../support/withV4';
+import BucketUtility from '../../lib/utility/bucket-util';
+import { WebsiteConfigTester } from '../../lib/utility/website-util';
+
+const bucketName = 'testdeletewebsitebucket';
+
```

Comment:
please add test when no such bucket and test when requester not authorized because not bucket owner.